amazon/aws-cli/container.yaml

```diff
@@ -0,0 +1,13 @@
+docker: amazon/aws-cli
+url: https://hub.docker.com/r/amazon/aws-cli
+maintainer: 'Amazon Web Services'
+description: The AWS Command Line Interface (AWS CLI) is a unified tool to manage your AWS services.
+latest:
+    2.13.0: sha256:e119d71e8a39a6a3dec5f60f06b090dcdb7adb1b463ae6e9c1cb55d52a05d418
+tags:
+    2.13.0: sha256:e119d71e8a39a6a3dec5f60f06b090dcdb7adb1b463ae6e9c1cb55d52a05d418
+    latest: sha256:e119d71e8a39a6a3dec5f60f06b090dcdb7adb1b463ae6e9c1cb55d52a05d418 
```

Comment:
I would avoid adding the `latest` tag in the list of tags. 
The `latest` key in the yaml serves the purpose.
Right, @vsoch ?

Reply:
yes!

Reply:
Thanks for this @marcodelapierre . I'll go over this again internally at Pawsey, let you review over there and then resubmit a new pull request into the original repo (here).